user-console/src/main/java/org/pentaho/mantle/client/admin/EmailAdminPanel.java

```diff
@@ -74,26 +90,53 @@ private Widget createEmailPanel() {
     vSpacer.setHeight( "20px" );
     mailPanel.add( vSpacer );
 
-    mailPanel.add( new Label( Messages.getString( "smtpHost" ) + ":" ) );
-    smtpHostTextBox = new TextBox();
-    smtpHostTextBox.setWidth( "220px" );
-    mailPanel.add( smtpHostTextBox );
+    HorizontalPanel protocolHbox = new HorizontalPanel();
+    protocolHbox.add( new Label( Messages.getString( "connectionProtocol" ) + ":" ) );
+    protocolHbox.getElement().setId( "protocolPanel" );
+
+    SimplePanel hSpacer = new SimplePanel();
+    hSpacer.setWidth( "15px" );
+    protocolHbox.add( hSpacer );
+
+    protocolsListBox = new ListBox();
+    protocolsListBox.addItem( Messages.getString( "smtp" ), "SMTP" );
+    protocolsListBox.addItem( Messages.getString( "smtps" ), "SMTPS" );
+    protocolsListBox.addItem( Messages.getString( "graphApiLabel" ), "graph_api" );
+    protocolHbox.add( protocolsListBox );
+    mailPanel.add( protocolHbox );
 
     vSpacer = new SimplePanel();
     vSpacer.setHeight( "10px" );
     mailPanel.add( vSpacer );
 
-    mailPanel.add( new Label( Messages.getString( "port" ) + ":" ) );
+    smtpPanel = new VerticalPanel();
+    smtpPanel.add( new Label( Messages.getString( "smtpHost" ) + ":" ) );
+    smtpHostTextBox = new TextBox();
+    smtpHostTextBox.setWidth( PANEL_WIDTH );
+    smtpPanel.add( smtpHostTextBox );
+
+    vSpacer = new SimplePanel();
+    vSpacer.setHeight( "10px" );
```

Comment:
10px is repeated, create java constant and 10 is  magic number, add javadoc comment as to why 10 was chosen